src/cli/rustup_mode.rs

```diff
@@ -918,6 +918,7 @@ fn check_updates(cfg: &Cfg) -> Result<utils::ExitCode> {
 
 fn update(cfg: &mut Cfg, m: &ArgMatches<'_>) -> Result<utils::ExitCode> {
     let self_update = !m.is_present("no-self-update") && !self_update::NEVER_SELF_UPDATE;
+    let forced = m.is_present("force-non-host");
```

Comment:
This will need a commensurate change in the Clap definitions earlier in the file, and in the error mesage below.